Sources/Deprecations+Removals.swift

```diff
@@ -207,6 +207,15 @@ extension SignalProtocol {
 
 	@available(*, unavailable, message: "This Signal may emit errors which must be handled explicitly, or observed using `observeResult(_:)`")
 	public func observeNext(_ next: (Value) -> Void) -> Disposable? { fatalError() }
+
+	@available(*, unavailable, renamed:"catch(_:)")
+	public func flatMapError<F>(_ handler: @escaping (Error) -> SignalProducer<Value, F>) -> Signal<Value, F> { fatalError() }
```

Comment:
`flatMapError` is far more descriptive. `catch` is a well known word but it doesn't carry the extra information that this does, which draws a parallel with `flatMap`.